sdk/js/src/token_bridge/getIsTransferCompleted.ts

```diff
@@ -209,3 +198,30 @@ export async function getIsTransferCompletedNear(
     })
   )[1];
 }
+
+export async function getIsTransferCompletedAptos(
+  client: AptosClient,
+  tokenBridgeAddress: string,
+  signedVAA: Uint8Array,
+): Promise<boolean> {
+  // get handle
+  tokenBridgeAddress = ensureHexPrefix(tokenBridgeAddress);
+  const state = (
+    await client.getAccountResource(tokenBridgeAddress, `${tokenBridgeAddress}::state::State`)
+  ).data as State;
+  const handle = state.consumed_vaas.elems.handle;
+
+  // check if vaa hash is in consumed_vaas
+  const signedVAAHash = await getSignedVAAHash(signedVAA);
+  try {
+    // when accessing Set<T>, key is type T and value is 0
+    await client.getTableItem(handle, {
+      key_type: "vector<u8>",
+      value_type: "u8",
+      key: signedVAAHash,
+    });
+    return true;
+  } catch {
```

Comment:
can we be a little more specific here about what specific exception this handles? if it's not the "table key missing" one, then I think the exception should be bubbled up instead of silently swallowed